src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.windows.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Platform;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Hosting;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+using WButton = Microsoft.UI.Xaml.Controls.Button;
+using WImage = Microsoft.UI.Xaml.Controls.Image;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+public partial class IconTintColorBehavior
+{
+	SpriteVisual? spriteVisual;
+	Vector2? originalImageSize;	
+
+	/// <inheritdoc/>
+	protected override void OnAttachedTo(View bindable, FrameworkElement platformView)
+	{
+		ApplyTintColor(platformView, bindable, TintColor);
+
+		bindable.PropertyChanged += OnElementPropertyChanged;
+		this.PropertyChanged += (s, e) =>
```

Comment:
should we unsubscribe to this event on detach?

Reply:
I had a similar thought, but then noticed that this matches the existing implementation in `IconTintColorBehavior.ios.cs`:

https://github.com/CommunityToolkit/Maui/blob/21cb4eac178627022790cff0af7c376977b2e81a/src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.ios.cs#L14-L20

Reply:
It's fine to not unsubscribe, since the event is related to the same object, then will have the same life time, so it will be collected.

To illustrate that imagine that you've a Page with a button on that page, then you subscribe to the Click event, if you use xaml, you can't unsubscribe that event and even with that the Page will not leak.

```csharp
class MyPage : ContentPage

Button loginButton = new();

public MyPage()
{
   loginButton.Clicked += (s,e) =>
  {
     // Do something
  }
}
```

Since the life time of Page and Button are the same, the event will be collected correctly.